src/datepicker/bs-datepicker.component.ts

```diff
@@ -69,8 +69,15 @@ export class BsDatepickerDirective implements OnInit, OnDestroy, OnChanges {
     if (this._bsValue === value) {
       return;
     }
-    this._bsValue = value;
-    this.bsValueChange.emit(value);
+    let date = value;
+
+    if (date) {
+      const timezoneOffset = value.getTimezoneOffset() * 60000;
```

Comment:
Is this possible to have a service to have a custom timezone offset?

Reply:
Maybe as different feature-request